openfl-tutorials/interactive_api/HPU/PyTorch_TinyImageNet/README.md

```diff
@@ -0,0 +1,140 @@
+# PyTorch_TinyImageNet
+
+## **How to run this tutorial (without TLC and locally as a simulation):**
+<br/>
+
+### 0. If you haven't done so already, create a virtual environment(Python>=3.8), install OpenFL, and upgrade pip:
```

Comment:
As I understand, Habana stack currently doesn't work well inside `venv`. Is this intended?

Reply:
Latest installation scripts it supports venv